src/pages/content/word-list.mdx

```diff
@@ -1,12 +1,707 @@
+
+### Zendesk Apps
+
+Capitalize this feature name: “The Zendesk Apps framework.”
+
+<hr />
+
+## Words we avoid
+
+These words aren’t bad. They’re just not in our terminology diet.
+
+### oops, whoops, oh no
+
+Oops, we did it again. Seriously, though, we don’t use oops or other
+self-deprecating blurts in error messages or empty states.
+
+### please
+
+That’s right. At Zendesk, you can skip the P-word. It takes up space in our UI
+and can come across as insincere.
+
+<BestPractice>
+  <Do>
+    <p>Your password was reset. Enter a new password.</p>
+  </Do>
+  <Dont>
+    <p>Your password was reset. Please enter a new password.</p>
+  </Dont>
+</BestPractice>
+
+### sorry
+
+We generally don’t apologize when things go wrong. When there’s an error, tell
+the user what happened and give them clear directions on what they can do next.
+
+Instead we save apologies for meaningful situations: when something goes really
+wrong or when Zendesk as a company has made a serious error, like deleting an
+account.
+
+### we
+
+Avoid we unless referring to a specific group of people. Don’t use we to refer
+to Zendesk the company or Zendesk the software. Similarly, avoid using our. See
+[Pronouns](/content/grammar#pronouns).
+
+<hr />
+
+## People
+
+The people in our Zendesk neighborhood. All of the people in the neighborhood
+have a job to do.
+
+### account owner
+
+An account owner is a type of administrator. The account name is associated with
+this person's name, usually the person who created the account. Just like The
+Highlander, there can be only one. For example, “Only the account owner has
+permission to make subscription changes.”
+
+### admin, administrator
+
+Admins have additional access to the Zendesk experience like setting up the
+workflows, adding users, and defining automations. We recommend admins instead
+of administrators. It’s friendlier and shorter. For example, “Allow admins to
+tag users.”
+
+Avoid telling agents to “talk to an admin.” Instead, inform them that an admin
+can do the task and leave the communication up to the agent. For example, “You
+don’t have Explore. An admin can activate that in your subscription.”
+
+### advisor
+
+In Support Enterprise, advisors manage the workflow and configure the Zendesk
+experience. Advisors don't solve tickets and they can only make private comments
+or add notes.
+
+### agent
+
+Agents are the bulk of the support staff for Zendesk subscribers. They’re
+assigned tickets and interact with their customers to resolve issues. For
+example, “Zendesk Support offers several tools to make your agents more
+efficient.”
+
+### customer
+
+Often synonymous with end user, but not all Zendesk subscribers think of their
+end users as customers. For example, the end users of an IT department are
+employees. Use subscriber to refer to people and organizations with a paid
+subscription to Zendesk.
+
+### editor
+
+Editors can create and customize new dashboards, queries, and datasets. (Explore
```

Comment:
_Editors_ can create and customize new dashboards, queries, and datasets. (Explore only.)